packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -143,8 +143,8 @@ export class NodeHttpRequestResponseHandler extends NodeHttpStreamsHandler {
         const contentTypeHeader = response.headers['content-type'];
 
         const charsetString = contentTypeHeader ? contentTypeHeader.split(';')
-          .filter((part) => part.includes('charset='))
-          .map((part) => part.split('=')[1].toLowerCase())[0]
+          .filter((part: string | string[]) => part.includes('charset='))
+          .map((part: string) => part.split('=')[1].toLowerCase())[0]
```

Comment:
Both types should be string arrays, I think.

Reply:
Second one should still be a string. You can't split on the array and if you split on part[0] that would just be an array containing ['c']. I tested this in the console and the jest tests also confirm.

Reply:
Ah, wow, was thinking in pipes ... so just the first one should definitely be an array, right?